src/scss/_variables.scss

```diff
@@ -191,3 +191,4 @@ $card-header-font-family: $font-medium;
 $card-subtitle-font-size: 0.875rem;
 $card-subtitle-line-height: 1.25rem;
 $card-subtitle-font-family: $font-medium;
+$card-hero-border-color: rgba(151, 151, 151, 0.12);
```

Comment:
@fuhton Good point. I'm reviewing the designs and it looks only that border is it's own color. Perhaps we should confirm that's intentional because all the other borders are `rgba(0, 0, 0, 0.12)`.